Taskfile.lint.yaml

```diff
@@ -81,7 +81,7 @@ tasks:
       - featurebyte/**/*
       - tests/**/*
     cmds:
-      - echo "${DARG_SOURCES}" | xargs -n16 -P {{ .CPU_CORES }} poetry run darglint
+      - echo "${DARG_SOURCES}" | xargs -n16 -P {{ .CPU_CORES }} poetry run darglint --docstring-style numpy --strictness full -v 2
```

Comment:
Is it expected that there isn't any docstrings that need to be fixed despite this change?

Reply:
I'm pretty surprised they didn't flag out any issues. It seems to be running on quite a number of files.

Reply:
Seems to be working as intended. Though it does not validate missing docstrings.